examples/dgmulti_1d/elixir_euler_shu_osher_gauss_shock_capturing.jl

```diff
@@ -0,0 +1,97 @@
+using Trixi
+using OrdinaryDiffEq
+
+γ_gas = 1.4
+equations = CompressibleEulerEquations1D(γ_gas)
+
+###############################################################################
+# setup the GSBP DG discretization that uses the Gauss operators from 
+# Chan, Del Rey Fernandez, Carpenter (2019). 
+# [https://doi.org/10.1137/18M1209234](https://doi.org/10.1137/18M1209234)
+
+# Shu-Osher initial condition for 1D compressible Euler equations
+# Example 8, https://www.sciencedirect.com/science/article/pii/0021999189902222
```

Comment:
Would you mind linking a doi to the article here instead of the ScienceDirect link?

Reply:
Corrected in the new commit